core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/columns/tree/TreeNode.kt

```diff
@@ -48,19 +48,34 @@ internal class TreeNode<T>(
         return addChild(childName, createData())
     }
 
-    fun dfs(enterCondition: (TreeNode<T>) -> Boolean = { true }, yieldCondition: (TreeNode<T>) -> Boolean = { true }): List<TreeNode<T>> {
+    @Deprecated("Use allChildren instead", ReplaceWith("allChildren(enterCondition, yieldCondition)"))
+    fun dfs(
```

Comment:
Why deprecate it? It's an internal class

Reply:
probably yes, but since it's used a lot around the code base and people (Anatoly) might be used to it, I thought it would be helpful to have a redirect.